test/loaders/test_video_loader.py

```diff
@@ -38,11 +39,60 @@ def create_sample_video(self):
                              dtype=np.uint8)
             out.write(frame)
 
-    def setUp(self):
-        self.create_sample_video()
+    def create_sample_video_with_similar_frames(self):
```

Comment:
Ahh... A few issues here. 

This is a PP filter, don't mix its tests with the loader. 

1. Create a new test file in `test/filters` and write your tests there. 
2. You are expected to write unit tests for your logic, so you wouldn't be needing simple video loader in your tests.
3. [For your new tests] Instead of creating a dummy video file with similar frames create a FrameBatch object and this will be input to your `predict` method. 

Thanks for updating setUp to setUpClass!

Reply:
Addressed all three comments. Tests for the added pp moved to test_frame_skipping_pp.py in tests/filters. Now passing batch of frames instead of creating a new video. @saiprashanth173